cpp/tensorrt_llm/thop/attentionOp.cpp

```diff
@@ -465,6 +466,8 @@ void attention_inplace(torch::Tensor q, torch::optional<torch::Tensor> k, torch:
     op->mRotaryEmbeddingOriginalMaxPositions = rotary_embedding_original_max_positions;
     op->mPagedContextFMHA = use_paged_context_fmha;
 
+    op->mAttentionChunkSize = attention_chunk_size;
```

Comment:
```
if(attention_chunekd_size.has_value())
{
  op->mAttentionChunkSize = attention_chunked_size.value();
}
else {
  // This might not be needed if you set the default values in common/attentionOp.h.
  op->mAttentionChunkedSize = INT_MAX;
}
```